### ISSUE TITLE
Consider creating a Gitter channel

### ISSUE THREAD

**Issue by tsm91** [ng-bootstrap/ng-bootstrap#441]
Hi, first of all let me say this is an awesome project. It seems like there will be a proper stable implementation of bootstrap in angular 2!

I realized nowadays i almost always ask questions via [Gitter](http://gitter.im/) instead of SO.

I think it would be a great idea to create an ng-bootstrap channel with an ng-bootstrap room, and in the future all bigger projects could have its own rooms.

**Comment by pkozlowski-opensource** [comment]
> Hi, first of all let me say this is an awesome project. It seems like there will be a proper stable implementation of bootstrap in angular 2!

Thnx! Indeed we aim at having top-notch impl of Bootstrap widgets for ng2. It _might_ look like we are moving more conservatively than other projects, but quality takes time!

> I realized nowadays i almost always ask questions via Gitter instead of SO.

So the downside is that responses to questions are not available to the wider community. The advantage of SO is that questions / answers can be searched / consulted "forever".

I'm not against creating a gitter channel as I think it is important to stay in touch with the community (especially in the early stages in the project) but I would tend to highly encourage using SO with the `ng-bootstrap` tag. I will monitor those and try to answer there.

tl;dr; Why not. Let's see what others got to say.

**Comment by icfantv** [comment]
We talked about this during our meeting and decided we were going to wait for now to see what the level of response would be.  Let's flag this as noted and keep an eye on it.

**Comment by icfantv** [comment]
Just an update here.  I've been on the Gitter for UIBS for about 6 months now and it's essentially gone unused save for a sporadic question here or there.  As of late, the bulk of the questions have nothing to do with Angular or our library, rather, they're about Bootstrap CSS.  Based on that, my initial inclination here is to reject this.

That said, we need to get the `ng-bootstrap` tag set up on SO (by creating a good question) so people have that venue.  Creating a new tag requires a reputation of 1500 or higher.

**Comment by sendilkumarn** [comment]
@icfantv Any slack channel available or any other medium to connect for development related discussions. If possible?

**Comment by tsm91** [comment]
> Any slack channel available or any other medium to connect for development related discussions. If possible?

Yeah i'm interested in this too. I would like to contribute to this and any bootstrap widget related ui development if i meet the requirements. I have a feeling i would learn a lot.

**Comment by pkozlowski-opensource** [comment]
Will put it on a back-burner for now. Happy to re-consider if there is more interest.

**Comment by sendilkumarn** [comment]
@pkozlowski-opensource Can we open a gitter at least? This will increase clarity and drive contributions. If it is okay, let me open an account and share it with you guys.

**Comment by intellix** [comment]
Would love a Gitter room for people to talk about this project. Especially since the Angular community is largely in Gitter

**Comment by icfantv** [comment]
@intellix. There still really hasn't been a ton of interest on this front.  Right now, the best method of support still stack overflow, recognizing that it's not immediate.

**Comment by intellix** [comment]
It's just that otherwise there isn't really a channel for people to discuss the library and it's components architecturally. I find SO just for basic issues like: "code not work, plzhalp". There are unofficial rooms already popping up. I wish gitter just opened rooms by default

**Comment by icfantv** [comment]
Yep, that's fair.  If we started it, however, I would say that it probably wouldn't be heavily monitored.
